frontend/.eslintrc.cjs

```diff
@@ -1,15 +1,32 @@
 module.exports = {
-    env: { browser: true, es2020: true },
-    extends: [
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
         "eslint:recommended",
-        "plugin:react/recommended",
-        "plugin:react/jsx-runtime",
-        "plugin:react-hooks/recommended"
+        "plugin:react/recommended"
```

Comment:
We need to make sure eslint and prettier do not contradict each other. If you see any problems, maybe try adding prettier here.
Example: 
https://github.com/BriptimusPrimus/sorted-list/blob/main/.eslintrc.js#L2